src/Components/Molecules/FMFormTextField/index.tsx

```diff
@@ -1,63 +1,79 @@
-import React, { memo } from 'react';
-import FormControl from '@mui/material/FormControl';
-import { FormHelperText, InputAdornment, InputLabel, Typography } from '@mui/material';
-import TextField from '@mui/material/TextField';
+import React, { memo } from "react";
+import FormControl from "@mui/material/FormControl";
+import {
+  FormHelperText,
+  InputAdornment,
+  InputLabel,
+  Typography,
+} from "@mui/material";
+import TextField from "@mui/material/TextField";
 
-import { IFMFormTextField } from './FMFormTextField.interface';
+import { IFMFormTextField } from "./FMFormTextField.interface";
 
 export const FMFormTextField = memo((props: IFMFormTextField) => {
-
-  const { testID, placeholder, type, label, helperText, onChange, adornment, adornmentPosition, value, MenuItems, ...materialUIProps } = props;
+  const {
+    testID,
+    placeholder,
+    type,
+    label,
+    helperText,
+    onChange,
+    adornment,
+    adornmentPosition,
+    value,
+    MenuItems,
+    ...materialUIProps
+  } = props;
 
   const AddAdornment = () => {
-    return (
-      <InputAdornment position="start">
-        {adornment}
-      </InputAdornment>
-    );
+    return <InputAdornment position="start">{adornment}</InputAdornment>;
   };
 
-  const startAdornment = adornment && adornmentPosition === 'start' ? (
-    AddAdornment()
-  ) : null;
+  const startAdornment =
+    adornment && adornmentPosition === "start" ? AddAdornment() : null;
 
-  const endAdornment = adornment && adornmentPosition === 'end' ? (
-    AddAdornment()
-  ) : null;
+  const endAdornment =
+    adornment && adornmentPosition === "end" ? AddAdornment() : null;
 
   return (
-    <div style={{ display: 'flex', alignItems: 'center', width: '100%', justifyContent: 'space-between' }}>
+    <div
+      style={{
+        display: "flex",
+        alignItems: "center",
+        width: "100%",
+        justifyContent: "space-between",
+      }}
+    >
       <Typography variant="subtitle1" gutterBottom component="div">
         {label}
       </Typography>
       <FormControl
-        sx={{ mt: 1, minWidth: '70%' }}
+        sx={{ mt: 1, minWidth: "70%" }}
         data-testid={testID}
         disabled
-        >
-        <InputLabel htmlFor="component-error">{type == 'date' || type == 'time' ? 'pohon' : ''}</InputLabel>
+      >
+        <InputLabel htmlFor="component-error">
+          {type == "date" || type == "time" ? "pohon" : ""}
+        </InputLabel>
         <TextField
```

Comment:
I think we just need to create `<FormControl>` and its helper text in molecules component, and a child form atom component (eg: TextField, Select, RadioButton, CheckBox, etc).

Reply:
beautiful!<3